cmd/podman/rm.go

```diff
@@ -14,7 +14,7 @@ var (
 	rmCommand     cliconfig.RmValues
 	rmDescription = fmt.Sprintf(`Removes one or more containers from the host. The container name or ID can be used.
 
-  Command does not remove images. Running containers will not be removed without the -f option.`)
+  Command does not remove images. Running containers will not be removed without the -f option. Containers with an invalid runtime configuration can be removed with the -ff option.`)
```

Comment:
Might be a learning for me, but I thought cobra would require a double dash `---ff` rather than a single `-ff`.

Reply:
Ah, I see below, CountVarP.